axelor-sale/src/main/java/com/axelor/apps/sale/web/CartController.java

```diff
@@ -97,4 +100,21 @@ public void createSaleOrder(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void getCartPartnerDomain(ActionRequest request, ActionResponse response) {
+    try {
+      Cart cart = request.getContext().asType(Cart.class);
+      List<CartLine> cartLineList = cart.getCartLineList();
+      String domain =
+          Beans.get(SaleOrderDomainService.class).getPartnerBaseDomain(cart.getCompany());
+
+      if (!(cartLineList == null || cartLineList.isEmpty())) {
+        domain = Beans.get(SaleOrderDomainService.class).getPartnerBaseDomain(cart.getCompany());
+      }
```

Comment:
You don't need this part of the code anymore. You are using the same method twice.